textattack/goal_functions/text/minimize_bleu.py

```diff
@@ -0,0 +1,47 @@
+import functools
+
+import nltk
+
+import textattack
+
+from .text_to_text_goal_function import TextToTextGoalFunction
+
+
+class MinimizeBleu(TextToTextGoalFunction):
+    """Attempts to minimize the BLEU score between the current output
+    translation and the reference translation.
+
+    Defined in It’s Morphin’ Time! Combating Linguistic Discrimination with
+    Inflectional Perturbations
+    (https://www.aclweb.org/anthology/2020.acl-main.263/).
+    """
+
+    EPS = 1e-10
+
+    def __init__(self, *args, target_bleu=0.0, **kwargs):
+        self.target_bleu = target_bleu
+        super().__init__(*args, **kwargs)
+
+    def _is_goal_complete(self, model_output, _):
+        bleu_score = 1.0 - self._get_score(model_output, _)
+        return bleu_score <= (self.target_bleu + MinimizeBleu.EPS)
+
+    def _get_score(self, model_output, _):
+        model_output_at = textattack.shared.AttackedText(model_output)
+        ground_truth_at = textattack.shared.AttackedText(self.ground_truth_output)
```

Comment:
Maybe override `init_attack_example` to save `self.ground_truth_at` so you don't have to cast it over and over again?

Reply:
eh, I don't think it's worth it :)